bitswap_test.go

```diff
@@ -178,7 +176,7 @@ func TestLargeSwarm(t *testing.T) {
 	if detectrace.WithRace() {
 		// when running with the race detector, 500 instances launches
 		// well over 8k goroutines. This hits a race detector limit.
-		numInstances = 50
+		numInstances = 37
```

Comment:
@Stebalien / @hannahhoward interestingly, this PR is failing due to too many go routines while running with the race detector. This error only shows up during the `TestLargeSwarm` test. The highest I seem to be able to set the  number of instances is `37`, but I haven't introduced any more go routines in this PR so I'm not sure yet why I have to do this. I'm going to keep looking into it, but thought I'd put a note here to see if either of you had an answer. Thanks.